### DISCUSSION TITLE
How to Clean install

### DISCUSSION THREAD

**Discussion by mkossigit** [vladmandic/sdnext#2357] [Q&A]
The new update recommends a clean install.
Can somebody tell me how to to this on my existing install?
I saw the command "Git clone" from vlad but not sure where to run this.
Does this mean all my settings and paths are removed?

**Comment by cgidesign-de** [comment]
E.g. on Windows:
Open a command prompt (type cmd in search to get it)

put:
git clone https://github.com/vladmandic/automatic \<optional directory name\>

into the command prompt.

\<optional directory name\> means you can define where sdnext should be installed - e.g.

git clone https://github.com/vladmandic/automatic f:/automatic

sdnext will be installed into f:/automatic

If you want to keep your existing install, choose a different location than the current one.
E.g.:
Your current install is in c:/automatic
Then e.g. use:
git clone https://github.com/vladmandic/automatic c:/automatic2

The new install will not use your existing settings. It is a new fresh new install.

**Reply by brknsoul** [reply]
Psst: Change your `/` to `\` ;-)

**Reply by cgidesign-de** [reply]
Yep, Windows would normaly be \ but command prompt supports both variants in my tests.

**Reply by mkossigit** [reply]
Many thanks!

**Comment by 3dfxgamer** [comment]
Question:
I have models and a lot of loras downloaded in the standard models directory, when performing git clone of the entire folder, they will be erased?.

**Reply by vladmandic** [reply]
clone to new folder. then either change model path to point to old or move them or symlink them.
